code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -976,6 +976,9 @@ GLOBAL_LIST_INIT(marine_selector_cats, list(
 		/obj/item/reagent_containers/hypospray/advanced = list(CAT_MEDSUP, "Advanced hypospray", 2, "black"),
 		/obj/item/reagent_containers/hypospray/advanced/big = list(CAT_MEDSUP, "Big hypospray", 10, "black"),
 		/obj/item/clothing/glasses/hud/health = list(CAT_MEDSUP, "Medical HUD glasses", 2, "black"),
+		/obj/item/clothing/glasses/hud/medgoggles = list(CAT_MEDSUP, "Medical HUD goggles", 2, "black"),
+		/obj/item/clothing/glasses/hud/medgoggles/prescription = list(CAT_MEDSUP, "Medical HUD goggles prescription", 2, "black"),
+		/obj/item/attachable/suppressor = list(CAT_ATT, "Suppressor", 0, "black"),
```

Comment:
Undocumented change

Reply:
i guess they mean the suppressor

Reply:
Suppressor has always been there, it showing as a change is probably from some me having to copy and past my addition in when I updated it against the master.

Reply:
Still do not know why this is showing as a change, point buy items I added were removed because of new ways to create the items. I'll be deleting this.

Reply:
I have looked back and it is not there. I do not know how the fuck I managed to stick that in there while copy pasting, but I'll remove it now.